R/board.R

```diff
@@ -132,6 +132,34 @@ board_cache_path <- function(name) {
   fs::path(path, name)
 }
 
+#' Deparse a board object
+#'
+#' Returns the R code that would recreate the board is re-run on another
+#' computer. Goal is to capture the location of the board, but not the
+#' authorisation, since (a) that leak would leak credentials and (b) in
+#' most deployment scenarios board auth will be read from env vars.
```

Comment:
```suggestion
#' Returns the R code that would recreate the board when re-run on another
#' computer. Goal is to capture the location of the board, but not the
#' authorisation, since (a) that would leak credentials and (b) in
#' most deployment scenarios board auth will be read from env vars.
```
I think??